.changeset/rare-pans-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": patch
```

Comment:
```suggestion
"nextjs-website": minor
```
IMHO this should be a minor, you are adding a new feature, you are not fixing anything broken or that does not work as expected